ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml

```diff
@@ -143,6 +143,18 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- always_run: false
```

Comment:
Shouldn't we remove the entry for `e2e-aws-gatewayapi` above?  Doesn't this replace it?

Reply:
I don't think the conformance test can replace existing `e2e-aws-gatewayapi`, the conformance is to test the implementation matches the API specification, but implementation specific features will not be covered by it. For example, `e2e-aws-gateway` covers the OSSM and DNSRecord tests but conformance doesn't.
We could run conformance tests as part of our full e2e test suite. see https://gateway-api.sigs.k8s.io/concepts/conformance/#conformance

Reply:
Thanks.  I misunderstood. Now I see the tests are completely different:
  - ref: ingress-gatewayapi-conf-gatewayclass refers to ci-operator/step-registry/ingress/gatewayapi/conf-gatewayclass/ingress-gatewayapi-conf-gatewayclass-ref.yaml
  - ref: ingress-gatewayapi-conformance-tests refers to ci-operator/step-registry/ingress/gatewayapi/conformance-tests/ingress-gatewayapi-conformance-tests-ref.yaml

This is a new pattern for us.  Normally we make very few code additions here because we keep test code in the repo it is used in.  I wonder what the benefit is to keep it in the step-registry?

Reply:
Yes, Miciah also recommend to add a Makefile target in openshift/cluster-ingress-operator for running the conformance tests, I'll try it later. If that works then we don't need keep it in the step-registry.

Currently to run conformance test with AWS we need:
1. ensure one gatewayclass is ready
2. update the timeout const "MaxTimeToConsistency" from 30s to 90s (otherwise some tests will fail)
I'm not sure if the 2nd one is acceptable since we need to update upstream code.

Reply:
@lihongan if we need to update upstream code to set the MaxTimeToConsistency we should start that soon.  Hopefully we could do that easily with a runtime flag, e.g. some examples in https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/utils/flags/flags.go.

Reply:
Currently we workaround it in the [hack/gatewayapi-conformance](https://github.com/openshift/cluster-ingress-operator/blob/edf5e71e8b08ef23a4d8f0b3fee5630c66625967/hack/gatewayapi-conformance.sh#L46-L48) of C-I-O repo, the original `ref` already removed from this PR.
Will improve it with more flexible way like using CustomTimeoutConfig.